portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,7 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+    response.setContentType("text/html;"); 
```

Comment:
Can you remove this file from the PR? The only change made is the whitespace.

Reply:
Will do.